ema_workbench/analysis/regional_sa.py

```diff
@@ -256,10 +258,18 @@ def plot_cdfs(x, y, ccdf=False):
     x = x.copy()
 
     try:
-        x = x.drop("scenario", axis=1)
+        x = x.drop("scenario_id", axis=1)
     except KeyError:
         pass
 
+    for entry in ["model", "policy"]:
+        if x.loc[:, entry].unique().shape != (1,):
+            continue
+        try:
+            x = x.drop(entry, axis=1)
+        except KeyError:
```

Comment:
Explicit warning needed?

Reply:
No, scenario_id, model, and policy are workbench defaults. to keep the code a bit more generic we remove these if they are available (and default to 1 category for model and policy).